src/features/groups/components/InviteMemberDialog/InviteMemberDialogContent.tsx

```diff
@@ -42,6 +43,7 @@ export const InviteMemberDialogContent = ({
     if (!result.ok) return;
 
     setIsLinkButtonClicked(true);
```

Comment:
Could you move this line to the top of the function? Currently, it's being called after `await`, and might cause delays, leading to users clicking the [copy] button multiple times.